### ISSUE TITLE
problems happened when using SQLCHECK

### ISSUE THREAD

**Issue by MoonforYoung** [microsoft/CSS_SQL_Networking_Tools#30]
it is strange that i start the sqlcheck.exe file in Windows terminal and do generate a log file. but got noting in it,what happened?

**Comment by Malcolm-Stewart** [comment]
This command worked in my Terminal using the default PowerShell command-line. Note in Powershell you have to prefix the app with a path, typically .\

  PS C:\Users\user1> cd \tools
  PS C:\tools> **.\**sqlcheck > sqlcheck.log
  PS C:\tools> notepad sqlcheck.log

Does it have the same behavior when run from cmd.exe?
Are you running it locally and not in a remote session?
Are you running "As Administrator"?
Please try running it without the pipe redirection. What shows in the console window?

**Comment by MoonforYoung** [comment]
already fixed.
I used a "start" command before the .exe file, just shown below. 
PS C:\tools> .\**start** sqlcheck > sqlcheck.log

I think that's why I can't get any output from the .log file cause a new window captured these outputs.
I tried to replace the "start" command with "&" in PowerShell, correct output did I get.

I think it's stupid for such a question to be posted. and thank you for your patience.
